src/overlays/actors/ovl_En_River_Sound/z_en_river_sound.c

```diff
@@ -27,15 +30,212 @@ const ActorInit En_River_Sound_InitVars = {
     (ActorFunc)EnRiverSound_Update,
     (ActorFunc)EnRiverSound_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_River_Sound/EnRiverSound_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_River_Sound/EnRiverSound_Destroy.s")
+void EnRiverSound_Init(Actor* thisx, GlobalContext* globalCtx) {
+    /**
+     * Params format:
+     *     0xPPBB
+     *     P: Sence path index
+     *     B: Behavior
+     **/
```

Comment:
Id move this comment over to the header for the time being. 
In a coming documentation pass we are going to have a standard way of documenting params. But until we have that format settled on, this is fine. 
But yea that documentation should live in the header since thats where people are going to look first to see the "external" things for the actor